video_recorder_node:
  ros__parameters:
    # Subscribers
    input_rgbd_image_topic: /camera/camera/rgbd

    # Parameters
    record_length: 60.0 # seconds
    output_dir: /home/user/RoboBounce/data